src/transformers/modeling_layoutlm.py

```diff
@@ -0,0 +1,905 @@
+# coding=utf-8
+# Copyright 2018 The Microsoft Research Asia LayoutLM Team Authors
```

Comment:
```suggestion
# Copyright 2018 The Microsoft Research Asia LayoutLM Team Authors and the HuggingFace Inc. team.
```

Reply:
I think you should use "Microsoft Corporation" instead of "MSRA LayoutLM authors". (Suggesting according to my own experience when I worked at MS) Maybe you can check with MS for this.

Reply:
@JetRunner Thanks for your suggestion!  I have checked with my leader and we decide to use the expression "MSRA LayoutLM authors".